specification/consumption/resource-manager/Microsoft.Consumption/2017-11-30/consumption.json

```diff
@@ -157,7 +157,21 @@
             "required": true,
             "type": "string",
             "enum": ["daily", "monthly"],
-            "modelAsString":true
+            "x-ms-enum": {
+              "name": "datagrain",
+              "modelAsString":true,
+              "values":[
+                        { 
+                          "value":"daily",
+                          "description":"Daily grain of data",
+                          "name":"DailyGrain"                          
+                        },
+                        { "value":"monthly",
```

Comment:
minor: if you could be consistent with formatting here, and adding a new line after the "{", it'd be great.

Reply:
Done